cves/CVE-2014-8547.yml

```diff
@@ -129,7 +128,7 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: 'During the lifetime of the vulnerability, most of the commits were about adding new features to the GIF decoder, such as supporting transparent pixels, and adding checks for edge cases and fixing issues with memory allocation, such as checking if the pointer is in a valid location, . Many developers worked on the file during the lifetime of the vulnerability, but one developer worked on the file for half a year before any other developers worked on the file. There was also a switch to refcounted AVframes, which is an API introduced in an updated version of a library they are using, as well as porting to the bytestream2 API. Also, near the end of the lifetime, a series of commits focused around fixing out of bounds vulnerabilities were pushed by the same developer that patched this vulnerability.'
```

Comment:
```suggestion
  answer: 'During the lifetime of the vulnerability, most of the commits were about adding new features to the GIF decoder, such as supporting transparent pixels, and adding checks for edge cases and fixing issues with memory allocation, such as checking if the pointer is in a valid location. Many developers worked on the file during the lifetime of the vulnerability, but one developer worked on the file for half a year before any other developers worked on the file. There was also a switch to refcounted AVframes, which is an API introduced in an updated version of a library they are using, as well as porting to the bytestream2 API. Also, near the end of the lifetime, a series of commits focused around fixing out of bounds vulnerabilities were pushed by the same developer that patched this vulnerability.'
```

Reply:
I find the first sentence a bit hard to read grammatically. One way you could reformat could be, 
"During the lifetime of the vulnerability, most of the commits were about adding new features to the GIF decoder. The commits pertained to various issues such as supporting transparent pixels, adding checks for edge cases, and fixing issues with memory allocation (such as checking if the pointer is in a valid location) ."